src/app/core/signing.service.ts

```diff
@@ -0,0 +1,85 @@
+import { Injectable } from '@angular/core';
+import { Fonts } from '../shared/interfaces';
+import { ActivatedRoute, Router } from '@angular/router';
+import { AuthService } from './auth.service';
+import { MsgIoService } from './msg-io.service';
+import { FontService } from './font.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SigningService {
+  public signingBoxText;
+  public backgroundColor;
+  public textColor;
+  public canSend;
+  public charCount;
+  public maxCharCount: number;
+  public charCountColor;
+
+  // font service stuff
```

Comment:
```suggestion
  // Font service variables
```
Let's be a bit more professional, shall we? ;P

Reply:
that wasn't me